### PR TITLE
sed: get nth line of a file

### PR DESCRIPTION
**Pull request by jamesob**
----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by sbrl** [comment]
Hey - thanks for the new page! Looks like you're having some issues with the linter. We've got the style we prefer documented over [here](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#markdown-format), and a few good examples are the pages for `tar`, `eyeD3`, and `ln`. If you need any assistance, just ask :smiley_cat:

**Comment by jamesob** [comment]
@sbrl thanks for checking in. I've fixed the linter error (AFAICT).

File: pages/common/sed.md [hunk 1]
```diff
@@ -29,3 +29,7 @@
 - Replace separator / by any other character not used in the find or replace patterns, e.g., #:
 
 `sed 's#{{find}}#{{replace}}#' {{filename}}`
+
+- Print only the `N`th line of a file:
```
Comment (agnivade) [inline pages/common/sed.md]: Let us remove backticks from N, and make it lower case. And to keep it consistent with other examples, use n for the example at line 23 also.

**Comment by mebeim** [comment]
Ping @jamesob :eyes:  it's been a while since the last comment. Care to address it so that this PR can be finally merged?

**Comment by mebeim** [comment]
Ping @agnivade LGTM

**Pull request merged**